data/beaconrestapi/src/main/java/tech/pegasys/artemis/beaconrestapi/schema/BeaconHeadResponse.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.artemis.beaconrestapi.schema;
+
+public class BeaconHeadResponse {
+  public final long slot;
+  public final String block_root;
+  public final String state_root;
+
+  public static Builder builder() {
+    return new Builder();
+  }
+
+  public static class Builder {
```

Comment:
I'm not sure I'd bother with a builder in this case.  It's a lot of extra boiler plate code for not a lot of value.  Just using the constructor directly is just as clear.  It also means that the compiler won't detect cases where fields are not being set which makes adding new fields in the future more error prone.